components/Rating.jsx

```diff
@@ -0,0 +1,3 @@
+const Rating = () => {
+  return <></>;
```

Comment:
While not necessarily broken, try to refrain from using `<></>` as an empty value. Use `null` to communicate "Nothing is returned".